modules/multi-architecture-creating-x86_64-bootimage.adoc

```diff
@@ -0,0 +1,137 @@
+----
+. Generate a shared access signature (SAS) token. Use this token to upload the {op-system} VHD to your storage container by running the following commands:
++
+[source,terminal]
+----
+$ end=`date -u -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
+----
++
+[source,terminal]
+----
+$ sas=`az storage container generate-sas -n ${CONTAINER_NAME} --account-name ${STORAGE_ACCOUNT_NAME} --https-only --permissions dlrw --expiry $end -o tsv`
+----
+. Copy the {op-system} VHD into the storage container by running the following command:
++
+[source,terminal]
+----
+$ az storage blob copy start --account-name ${STORAGE_ACCOUNT_NAME} --sas-token "$sas" \
+ --source-uri "${RHCOS_VHD_ORIGIN_URL}" \
+ --destination-blob "${BLOB_NAME}" --destination-container ${CONTAINER_NAME}
+----
++
+You can check the status of the copying process by running the following command:
++
+[source,terminal]
+----
+$ az storage blob show -c ${CONTAINER_NAME} -n ${BLOB_NAME} --account-name ${STORAGE_ACCOUNT_NAME} | jq .properties.copy
+----
++
+.Example output
+[source,terminal]
+----
+{
+ "completionTime": null,
+ "destinationSnapshot": null,
+ "id": "1fd97630-03ca-489a-8c4e-cfe839c9627d",
+ "incrementalCopy": null,
+ "progress": "17179869696/17179869696",
+ "source": "https://rhcos.blob.core.windows.net/imagebucket/rhcos-411.86.202207130959-0-azure.aarch64.vhd",
+ "status": "success", <1>
+ "statusDescription": null
+}
+----
++
+<1> If the `status` parameter displays the `success` object, the copying process is complete.
+
+. Create an image gallery by running the following command:
++
+[source,terminal]
+----
+$ az sig create --resource-group ${RESOURCE_GROUP} --gallery-name ${GALLERY_NAME}
+----
+Use the image gallery to create an image definition by running the following command:
++
+[source,terminal]
+----
+$ az sig image-definition create --resource-group ${RESOURCE_GROUP} --gallery-name ${GALLERY_NAME} --gallery-image-definition rhcos-x86_64 --publisher RedHat --offer x86_64 --sku x86_64 --os-type linux --architecture x64 --hyper-v-generation V2
+----
++
+In this example command, `rhcos-x86_64` is the name of the image definition.
+. To get the URL of the VHD and set it to `RHCOS_VHD_URL` as the file name, run the following command:
++
+[source,terminal]
+----
+$ RHCOS_VHD_URL=$(az storage blob url --account-name ${STORAGE_ACCOUNT_NAME} -c ${CONTAINER_NAME} -n "${BLOB_NAME}" -o tsv)
+----
+. Use the `RHCOS_VHD_URL` file, your storage account, resource group, and image gallery to create an image version by running the following command.
++
+[source,terminal]
+----
+$ az sig image-version create --resource-group ${RESOURCE_GROUP} --gallery-name ${GALLERY_NAME} --gallery-image-definition rhcos-arm64 --gallery-image-version 1.0.0 --os-vhd-storage-account ${STORAGE_ACCOUNT_NAME} --os-vhd-uri ${RHCOS_VHD_URL}
+----
++
+In this example, `1.0.0` is the image version.
+. Your `x86_64` boot image is now generated. You can access the ID of your image by running the following command:
```

Comment:
Same comment as above about the first sentence being written in the imperative mood. Here's one way to rewrite this but keep all of the same information:
```suggestion
. Access the ID of the generated `x86_64` boot image by running the following command:
```

If the "you can" is meant to indicate that this is optional, I'd add that at the beginning of the step:
```suggestion
. Optional: Access the ID of the generated `x86_64` boot image by running the following command:
```

Reply:
Updated as per the suggestion.